app/defines.php

```diff
@@ -11,10 +11,10 @@
 namespace UserFrosting;
 
 // Some standard defines
-define('UserFrosting\VERSION', '4.5.0');
+define('UserFrosting\VERSION', '4.6.0-dev');
 define('UserFrosting\DS', '/');
-define('UserFrosting\PHP_MIN_VERSION', '^7.2');
-define('UserFrosting\PHP_RECOMMENDED_VERSION', '^7.4');
+define('UserFrosting\PHP_MIN_VERSION', '>=7.4');
```

Comment:
I suggest `'^7.4 || ^8.0'`.

Reply:
Is it `^7.4 || ^8.0` Or `^7.4|^8.0` ?

Reply:
`^7.4 | ^8.0`, sorry I've been programming in JavaScript for a month straight. Even with the aid of TypeScript, it does things to the mind...